modules/arkit/arkit_interface.mm

```diff
@@ -582,28 +582,28 @@
 
 					// copy our current frame transform
 					matrix_float4x4 m44 = camera.transform;
-					if (orientation == UIDeviceOrientationLandscapeLeft) {
+					if (orientation == UIInterfaceOrientationLandscapeLeft) {
```

Comment:
As I understand the docs the logic for `LandscapeLeft` and `LandscapeRight` should be swapped for `UIInterfaceOrientation` vs `UIDeviceOrientation`: https://developer.apple.com/documentation/uikit/uiinterfaceorientation

Yet the constants have the same value and I think here we were implicitly casting `UIInterfaceOrientation` values to `UIDeviceOrientation`, so the logic should already be correct as is?